public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -178,7 +178,7 @@ export interface ModalProps {
   hideFolder?: boolean;
 }
 
-export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
+export function EditRuleGroupModal(props: ModalProps): React.ReactElement {
```

Comment:
I changed the name, as this group modal is not for cloud?🤔

Reply:
Perhaps a left-over from some refactoring? :D